src/CMakeLists.txt

```
@@ -661,6 +665,30 @@ target_link_libraries(MIOpen PRIVATE ${CMAKE_THREAD_LIBS_INIT} ${BZIP2_LIBRARIES
 generate_export_header(MIOpen
     EXPORT_FILE_NAME ${PROJECT_BINARY_DIR}/include/miopen/export.h
 )
+
+if(MIOPEN_ENABLE_AI_KERNEL_TUNING)
+        find_path(FDEEP_INCLUDE_DIR "fdeep/fdeep.hpp")
+        find_path(EIGEN_INCLUDE_DIR "eigen3/Eigen/Core")
+        if(${FDEEP_INCLUDE_DIR} STREQUAL "FDEEP_INCLUDE_DIR-NOTFOUND")
+            message(WARNING "MIOPEN_ENABLE_AI_KERNEL_TUNING requirement frugally-deep not found, setting MIOPEN_ENABLE_AI_KERNEL_TUNING to FALSE")
+            set(MIOPEN_ENABLE_AI_KERNEL_TUNING FALSE)
```

In this case processing should be stopped
```suggestion
            message(FATAL_ERROR "MIOPEN_ENABLE_AI_KERNEL_TUNING requirement frugally-deep not found")
```

---

@averinevg This is duplicate of https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2031#discussion_r1153837429